packages/common-all/src/sidebar.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("GIVEN sidebars object", () => {
```

Comment:
for tests, can we follow [[dendron://dendron.docs/dev.process.qa.test#writing-tests]]? tests for common-all should be in `engine-test-utils`

Reply:
[`9166a8b` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/9166a8bd20af8eafc6435c918a5f4a59e887aeba)